pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -95,3 +96,83 @@ func (ac *OSSAccessControlService) GetUserBuiltInRoles(user *models.SignedInUser
 
 	return roles
 }
+
+func (ac *OSSAccessControlService) saveFixedRole(role accesscontrol.RoleDTO) {
+	if storedRole, ok := accesscontrol.FixedRoles[role.Name]; ok {
+		// If a package wants to override another package's role, the version
+		// needs to be increased. Hence, we don't overwrite a role with a
+		// greater version.
+		if storedRole.Version >= role.Version {
+			return
```

Comment:
It might be helpful to users if we warn them about which roles weren't updated?  Or maybe which ones were?

And what do you feel about cancelling the entire load if even one role doesn't update?

Reply:
Agree with you, I added a log line.
I'm going to open an issue for the behavior we want to have in case one role does not update :slightly_smiling_face:

Reply:
https://github.com/grafana/grafana-enterprise/issues/1716